recipes/libxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibxlsxwriterConan(ConanFile):
+    name = "libxlsxwriter"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jmcnamara/libxlsxwriter"
+    topics = ("conan", "Excel", "XLSX")
+    description = "A C library for creating Excel XLSX files"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_system_minizip": [True, False]
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+        "use_system_minizip": False,
+    }
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.options.use_system_minizip:
+            self.options["zlib"].minizip = True
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("zlib/1.2.11")
```

Comment:
Is this the minizip dependency?
We also have `minizip/1.2.11`.

(this  suggestion is UNTESTED)

https://github.com/jmcnamara/libxlsxwriter/blob/ff35a56934f7bb528f8f3d35debbd4569628a50d/CMakeLists.txt#L186-L194
```suggestion
        self.requires("zlib/1.2.11")
        self.requires("minizip/1.2.11")
```

Reply:
done